.circleci/config.yml

```diff
@@ -116,9 +123,18 @@ workflows:
             tags:
               only: /.*/
       - harness:
+          name: harness-macos
+          executor: macos
           filters:
             tags:
               only: /.*/
+      - harness:
+          executor: docker-node
+          filters:
+            branches:
+              only: master
```

Comment:
I guess you might want to include develop here later on.

Reply:
I'll do whatever I can to *not* use git flow in Prism. It's solving problems that Prism does not have.